test/e2e/batcher/test_batcher_custom_port.py

```diff
@@ -0,0 +1,85 @@
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import json
+from kubernetes import client
+
+from kserve import KServeClient
+from kserve import constants
+from kserve import V1beta1PredictorSpec
+from kserve import V1beta1Batcher
+from kserve import V1beta1SKLearnSpec
+from kserve import V1beta1InferenceServiceSpec
+from kserve import V1beta1InferenceService
+
+from kubernetes.client import V1ResourceRequirements, V1ContainerPort
+from ..common.utils import predict_str
+from ..common.utils import KSERVE_TEST_NAMESPACE
+from concurrent import futures
+
+kserve_client = KServeClient(config_file=os.environ.get("KUBECONFIG", "~/.kube/config"))
+
+
+input_file = open('./data/iris_batch_input.json')
+json_array = json.load(input_file)
+
+
+def test_batcher_custom_port():
+    service_name = 'isvc-sklearn-batcher-custom'
+
+    predictor = V1beta1PredictorSpec(
+        batcher=V1beta1Batcher(
+            max_batch_size=32,
+            max_latency=5000,
+        ),
+        min_replicas=1,
+        sklearn=V1beta1SKLearnSpec(
+            args=["--http_port=5000"],
+            storage_uri="gs://kfserving-samples/models/sklearn/iris",
```

Comment:
we have upgraded the sklearn version, so we should use `gs://kfserving-examples/models/sklearn/1.0/model` now.